server/celery_config.py

```
@@ -170,6 +171,12 @@
 except ImportError as e:
     logging.warning(f"Failed to import PagerDuty tasks: {e}")
 
+try:
+    import routes.victorops.tasks  # noqa: F401
+    logging.info("Splunk On-Call (VictorOps) tasks imported successfully")
+except ImportError as e:
+    logging.warning(f"Failed to import VictorOps tasks: {e}")
```

Ruff (0.15.12)

[warning] Logging statement uses f-string

(G004)